src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/Offender.java

```diff
@@ -90,31 +90,31 @@
     @Column(name = "ROW_VERSION")
     private Long rowVersion;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
All these were eager cause several selects for each retrieve for an offender even when these fields were never returned

Reply:
wow! Didn't realise these were all being EAGER loaded!